docs/source/examples/substrafl/go_further/run_iris_sklearn.ipynb

```diff
@@ -197,9 +197,9 @@
         "import numpy as np\n",
```

Comment:
<p>see also the <code>predict</code> bullet point that is outdated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/376/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>